File: Python/Fundamentals/Functions/Exercise_Functions/loading_bar.py
def bar_values(value):
    if value == 100:
        return f'100% Complete!\n[%%%%%%%%%%]'

    return f'{value}% [{(value // 10) * "%"}{(10 - value // 10) * "."}]\nStill loading...'


number = int(input())
print(bar_values(number))

File: Python/JS-Front-End/Functions-and-statements/Exercise/loading-bar.js
function solve(number) {
    if (number === 100) {
        console.log('100% Complete!');
    } else {
        repeatNum = number / 10
        console.log(`${number}% [${'%'.repeat(repeatNum)}${'.'.repeat(10 - repeatNum)}]`);
        console.log('Still loading...');
    }

}

solve(30);